civis/io/_tables.py

```diff
@@ -918,6 +931,10 @@ def civis_file_to_table(file_id, database, table, client=None,
         The column to use as the sortkey for the table.
     sortkey2 : str, optional
         The second column in a compound sortkey for the table.
+    table_columns : list[Dict[str, str]], optional
```

Comment:
Is a useful error message going to come back if this is incorrectly specified?

Reply:
The import will fail if this is the case.

Reply:
Will it fail with an informative error message?  I'd prefer a useful error message from platform, but if that's not available, then it seems like input validation here might be warranted.

Reply:
the import will fail with saying the table could not be created due to the bad sqlType provided example: `Job::UserError: Failed to create table: type "json" does not exist`

Reply:
Hmm, that's not particularly helpful but better than just "your job failed".  I'll follow up internally about maybe filing an internal issue about improving that error message from platform. I'm OK with leaving this as is.

Reply:
If i'm reading this error correctly, a "json" data type was provided in `table_columns` but its not a valid data type on Redshift?

Reply:
This is the error returned by Redshift. The API simply returns the error the database provides. Creating and maintaining some complex mapping of Redshift/Postgres errors to what we consider more user friendly errors is not something we have resourcing for right now.

If there are other scenarios that are providing unhelpful errors due to validation we are doing in our API then I'm all for addressing that.

Reply:
I thought you meant, sqlType was incorrectly specified, I can add a validation on if it's a list.